### ISSUE TITLE
how much time will this code cost?

### ISSUE THREAD

**Issue by Chuan1997** [linzh92/DeepICF#6]
it seems that this code runs forever on my cpu...

**Comment by linzh92** [comment]
> it seems that this code runs forever on my cpu...

You can try to set the value of parameter of `epochs` or stop running codes manully when the training loss is converging.
